app/src/organisms/ErrorRecoveryFlows/hooks/useRecoveryCommands.ts

```diff
@@ -365,6 +405,15 @@ export function useRecoveryCommands({
     }
   }, [chainRunRecoveryCommands, unvalidatedFailedCommand])
 
+  const manualRetrieve = useCallback((): Promise<CommandData[]> => {
+    const manualRetrieveCommand = buildManualRetrieve(unvalidatedFailedCommand)
+    if (manualRetrieveCommand == null) {
+      return Promise.reject(new Error('Invalid use of manual retrieve command'))
```

Comment:
```suggestion
      return reportAndRouteFailedCmd(new Error('Invalid use of manual retrieve command'))
```
See comments on https://github.com/Opentrons/opentrons/pull/18098. This is a rather new change, but we don't want to `Promise.reject` here.

This will need to change for `releaseLabwareLatch`, `closeLabwareLatch`, `homeShuttle`, too.